js/private/js_run_devserver.mjs

```diff
@@ -48,15 +48,36 @@ function mkdirpSync(p) {
 // pointing to the source symlink.
 async function syncRecursive(src, dst, writePerm) {
     try {
-        const lstat = await fs.promises.lstat(src)
+        const isWindows = process.platform === 'win32'
+        if (isWindows && path.basename(src) == 'node_modules') {
+            // special handling on Windows for node_modules
```

Comment:
Can we explain what that handling is and why? Why it's different then the non-windows special handling?